Sources/Commands/PackageTools/InstalledPackages.swift

```diff
@@ -0,0 +1,127 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import CoreCommands
+import TSCUtility
+import TSCBasic
+import PackageModel
+import Foundation
+
+extension SwiftPackageTool {
+    struct Install: SwiftCommand {
+        static var configuration: CommandConfiguration {
+            CommandConfiguration(abstract: "Offers the ability to install executable products of the current package.")
```

Comment:
Please add a `name` argument here specifying `experimental-install`, same for the uninstall subcommand. Until we have an accepted Swift Evolution proposal for this it has to have an `experimental-` prefix.

Reply:
There